libraries/GCS_MAVLink/GCS_MAVLink_Parameters.cpp

```diff
@@ -202,6 +210,18 @@ const AP_Param::GroupInfo GCS_MAVLINK::var_info[] = {
     // @RebootRequired: True
     // @User: Advanced
     AP_GROUPINFO("_ADSB",   10, GCS_MAVLINK, streamRates[GCS_MAVLINK::STREAM_ADSB], DRATE(GCS_MAVLINK::STREAM_ADSB)),
+
+    // ------------
+    // IMPORTANT: Add new stream rates *before* the _OPTIONS parameter.
+    // ------------
+
+    // @Param: _OPTIONS
+    // @DisplayName: Bitmask for configuring this telemetry channel
+    // @Description: Bitmask for configuring this telemetry channel. For having effect on all channels, set the relevant mask in all SRx_OPTIONS parameters. Keep in mind that part of the flags may require a reboot to take action.
+    // @RebootRequired: True
+    // @User: Standard
+    // @Bitmask: 0:Enable MavLink2 signing (Reboot required)
+    AP_GROUPINFO("_OPTIONS",   11, GCS_MAVLINK, options, float(AP_MAV_DEFAULT_STREAM_RATE_OPTIONS)),
```

Comment:
```suggestion
    AP_GROUPINFO("_OPTIONS",   20, GCS_MAVLINK, options, 0),
```

... leave a gap

We can't really have non-zero defaults for bitmasks.  Or at least not fior the first bits we set when we create the parameter.  So we might as well be consistent - leave the default value as zero.

It sucks talking in double-negatives, but there you go.